Sources/Testing/Issues/Issue.swift

```diff
@@ -103,6 +103,20 @@ public struct Issue: Sendable {
   /// The severity of this issue.
   @_spi(Experimental)
   public var severity: Severity
+  
+  /// Whether or not this issue should cause the test it's associated with to be
+  /// considered a failure.
+  ///
+  /// The value of this property is `true` for issues which have a severity level of
+  /// ``Issue/Severity/error``` or greater and are not known issues via ``withKnownIssue(_:...)``.
```

Comment:
Can you Build Documentation locally to ensure this DocC symbol reference resolves successfully? I intentionally put `...` (ellipsis) so I know it isn't valid as written